src/fire.py

```diff
@@ -0,0 +1,43 @@
+import vtk

```

Comment:
According to [the official docs](https://vtk.org/doc/nightly/html/md__builds_gitlab_kitware_sciviz_ci_Documentation_Doxygen_PythonWrappers.html), we should import with `from vtkmodules` instead of the deprecated `import vtk` for better performance.